src/workspaces/features/SidebarMenu/SidebarMenu.tsx

```diff
@@ -224,6 +234,28 @@ const SidebarMenu = (props: SidebarMenuProps) => {
                   <span className="text-sm leading-tight tracking-tight">
                     {ws.name}
                   </span>
+                  {ws.permissions.delete && (
```

Comment:
Is this permission enough ? This is one currently used in the settings panel but it seems weird that we only check for the workspace permissions and not the user permission. I lack some context here